src/modules/paste.ts

```diff
@@ -31,18 +31,12 @@ export interface PasteOptions {
 
 export function paste(editor: Editor, options?: PasteOptions) {
 
-  function onPaste(event: ClipboardEvent) {
-    if (!editor.enabled || !editor.doc.selection) return;
-    event.preventDefault();
-    const dataTransfer = event.clipboardData;
+  function paste(text: string, html: string) {
```

Comment:
Great work!

Perhaps a `selection?: EditorRange ` can be added as a 3rd parameter so that one might be able to paste into a document without the user's explicit selection.
```ts
selection = selection || doc.selection;
selection = selection && normalizeRange(doc.selection);
```

And since we have 3 parameters, and they could all be optional, perhaps we should use an options object instead.

```ts
interface PasteOptions {
  text?: string;
  html?: string;
  selection?: string;
}
```

Reply:
updated per your suggestions